IssueTracker/DataAccess/DbAccess/SQLDataAccess.cs

```diff
@@ -3,9 +3,13 @@
 using Dapper;
 using System.Data;
 using DataAccess.Utils;
+using System.Reflection;
 
 namespace DataAccess.DbAccess
 {
+#pragma warning disable CS8600 // Converting null literal or possible null value to non-nullable type.
```

Comment:
remove these lines, and fix the warnings

Reply:
I will try to solve this.